src/discretize.jl

```diff
@@ -401,7 +401,7 @@ to the PDE.
 For more information, see `discretize` and `PINNRepresentation`.
 """
 function SciMLBase.symbolic_discretize(pde_system::PDESystem,
-                                       discretization::PhysicsInformedNN)
+    discretization)
```

Comment:
This is type piracy. It needs to specify the top level dispatch.

Reply:
missed out on this, thanks